src/mongocxx/test/spec/unified_tests/runner.cpp

```diff
@@ -813,51 +858,54 @@ void run_tests_in_file(const std::string& test_path) {
     }
 }
 
-TEST_CASE("unified format spec automated tests", "[unified_format_spec]") {
-    instance::current();
+// Check the environment for the specified variable; if present, extract it
+// as a directory and run all the tests contained in the magic "test_files.txt"
+// file:
+bool run_unified_format_tests_in_env_dir(const std::string& env_path) {
+    const char* p = std::getenv(env_path.c_str());
 
-    std::string path = std::getenv("UNIFIED_FORMAT_TESTS_PATH");
-    CAPTURE(path);
-    REQUIRE(path.size());
+    if (nullptr == p) 
+        WARN("unable to look up path from environment variable \"" << env_path << "\"");
 
-    std::ifstream files{path + "/test_files.txt"};
-    REQUIRE(files.good());
+    CAPTURE(env_path);
+    REQUIRE(nullptr != p);
```

Comment:
`WARN`/`CAPTURE` + `REQUIRE` is redundant. I recommend instead:
```cpp
if (!p) {
    FAIL("missing required environment variable \"" << env_path << "\"");
}
```

Reply:
That would be nice! Is FAIL() available in the public API via the single header we're using? I see it in the file, but it doesn't appear to be documented.

Reply:
`FAIL` is documented under [streaming macros](https://github.com/catchorg/Catch2/blob/devel/docs/logging.md#streaming-macros).

Reply:
Oh, they're /hiding/ it!! Thank you.

Reply:
This suggestion has not yet been implemented.